project/Build.scala

```diff
@@ -574,6 +574,11 @@ object Build {
       // Add git-hash used to package the distribution to the manifest to know it in runtime and report it in REPL
       packageOptions += ManifestAttributes(("Git-Hash", VersionUtil.gitHash)),
 
+      scalacOptions ++= Seq(
+        "-Xfatal-warnings",
```

Comment:
I would enable this on all projects but only on the CI, when doing local development warnings as errors is just annoying.

Reply:
Good idea